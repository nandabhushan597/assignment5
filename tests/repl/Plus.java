// Tests for +
3 + 4
  :t 3 + 4
    3 + 4.
    :t 3 + 4.
      :t 3 + 4.f
	1.2f + 1.2f + 1.2f
	:t 3 + 4l + 5d
	  "hi" + 4
	  4 + "hi"
	  :t 4 + "hi"
	  4 + 5 + "hi"
	  4 + "hi" + 5
	  "hi" + 4 + 5
	  null + 4
	  4 + null
	  "hi" + null
	  null + "hi"
	  null + null
	  :t null + null
	  true + false
	  :t true + false
