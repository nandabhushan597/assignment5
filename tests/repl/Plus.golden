Exit code: ExitSuccess
Stdout:
Welcome to the Java interpreter.
Type `:?` at the prompt for the list of commands.
Java> Java> 7
Java> int
Java> 7.0
Java> double
Java> float
Java> 3.6000001f
Java> double
Java> @4 => "hi4"
Java> @8 => "4hi"
Java> String
Java> @12 => "9hi"
Java> @18 => "4hi5"
Java> @24 => "hi45"
Java> <nulltype> is not convertible to a numeric type.
Java> <nulltype> is not convertible to a numeric type.
Java> @28 => "hinull"
Java> @32 => "nullhi"
Java> <nulltype> is not convertible to a numeric type.
Java> <nulltype> is not convertible to a numeric type.
Java> boolean is not convertible to a numeric type.
Java> boolean is not convertible to a numeric type.
Java> Good-bye.

Stderr:
